packages/core/test/lib/integrations/console.test.ts

```diff
@@ -0,0 +1,75 @@
+import { describe, it, expect, vi, beforeEach } from 'vitest';
+import { captureConsoleBreadcrumb } from '../../../src/integrations/console';
+import { addBreadcrumb } from '../../../src/breadcrumbs';
+
+vi.mock('../../../src/breadcrumbs', () => ({
+  addBreadcrumb: vi.fn(),
+}));
+
+describe('captureConsoleBreadcrumb', () => {
+  beforeEach(() => {
+    vi.clearAllMocks();
+  });
+
+  it('creates a breadcrumb with correct properties for basic console log', () => {
+    const level = 'log';
+    const args = ['test message', 123];
+
+    captureConsoleBreadcrumb(level, args);
+
+    expect(addBreadcrumb).toHaveBeenCalledWith(
+      expect.objectContaining({
+        category: 'console',
+        data: {
+          arguments: args,
+          logger: 'console',
+        },
+        level: 'log',
+        message: 'test message 123',
+      }),
+      {
+        input: args,
+        level,
+      },
+    );
+  });
+
+  it('handles different console levels correctly', () => {
+    const levels = ['debug', 'info', 'warn', 'error'] as const;
```

Comment:
l/nit: I'd rather use `it.each` so that we'd know for which specific level the test failed. 

Also q: I noticed this pattern in quite a few tests (not yours, more generally in the repo) and was wondering: Am I maybe missing something why this is preferrable over `it.each`? 😅  

Again, not a blocker, feel free to ignore

Reply:
> Also q: I noticed this pattern in quite a few tests (not yours, more generally in the repo) and was wondering: Am I maybe missing something why this is preferrable over it.each? 😅

I've been leaning less toward using `it.each` because it's easier for me to `it.only` test one at a time, but I agree the table test is the better pattern here. Will change.